src/router/router-guards/itemMeasurements.js

```diff
@@ -7,8 +7,13 @@ export default async function itemMeasurements(to, from, next) {
   const { activeItem } = state;
 
   try {
-    if (!state.selectedPeriod) {
-      const periods = getPeriods();
+    const periods = getPeriods();
+
+    /*
+     * TODO: The second test here is only temporary and can be removed once the
+     * period selectors on the OKR and KPI pages have been reconciled.
+     */
+    if (!state.selectedPeriod || !periods[state.selectedPeriod]) {
```

Comment:
Er hensikten med sistnevnte test å ikke resette dersom man f.eks. har besøkt målinger, valgt en periode, går til "om", og tilbake til målinger? I så fall; fungerer det? 😅  Skjønner den skal det om den har vært innom "OKR" og fått en periode som ikke gir mening for målinger der.